guide/using-python.md

```diff
@@ -21,15 +17,15 @@ Webots starts Python using the standard `python` command line.
 As a consequence, it executes the first `python` binary found in the current `PATH`.
 If you want to use a different version of Python, please install it if needed and configure your environment so that it becomes the default `python` version when called from the command line in a terminal.
 Alternatively, you can change the default Python command from the Webots Preferences in the General tab.
-If you set it for example to `python3.6` instead of `python`, this version of python will be used (if available from the command line). Finally, it is also possible to set a different version of Python for each robot controller by editing the `[python]` section of the `runtime.ini` file in each robot controller directory and setting the `COMMAND` value to `python3`, `python3.6` or `python2.7`, etc.
-If specified in the `runtime.ini` file of a controller, this Python command will be executed instead of the default one to launch this controller.
+If you set it for example to `python3.6` instead of `python`, this version of python will be used (if available from the command line).
+Finally, it is also possible to set a different version of Python for each robot controller by editing the `[python]` section of the `runtime.ini` file in each robot controller directory and setting the `COMMAND` value to `python3`, `python3.6` or `python2.7`, etc. If specified in the `runtime.ini` file of a controller, this Python command will be executed instead of the default one to launch this controller.
```

Comment:
This one looks wrong (not sure if it is a merge conflicts issue or if it was already the case).

Reply:
It is wrong on master:

https://raw.githubusercontent.com/omichel/webots-doc/master/guide/using-python.md

I'm doing a PR targeting master to fix this.